lang/java/avro/src/test/java/org/apache/avro/specific/TestSpecificRecordWithUnion.java

```diff
@@ -29,7 +29,8 @@
 import org.apache.avro.io.DatumWriter;
 import org.apache.avro.io.BinaryEncoder;
 import org.apache.avro.io.Decoder;
-import org.junit.Test;
+
+import org.junit.jupiter.api.Test;
```

Comment:
Usually, you also rename the test methods (and remove the `public` keyword). Why not here?